src/features/SendAssets/components/SendAssetsForm.tsx

```diff
@@ -868,3 +868,5 @@ export const SendAssetsForm = ({ txConfig, onComplete, protectTxButton }: ISendF
     </div>
   );
 };
+
+export default SendAssetsForm;
```

Comment:
Why?

Reply:
Oh I did this when I removed the connect from redux

Reply:
Oh I did this when I removed the redux connect, I can change it to a regular import